app/src/main/java/au/com/shiftyjelly/pocketcasts/ui/MainActivity.kt

```diff
@@ -694,16 +692,11 @@ class MainActivity :
         }
     }
 
-    private fun updateNavAndStatusColors(playerOpen: Boolean, playingPodcast: Podcast?) {
+    private fun updateNavAndStatusColors(playerOpen: Boolean) {
```

Comment:
There is no need to pass the podcast as we don't colour the navigation bar as the fragments will now do this.